crypto/ec/build.info

```diff
@@ -125,3 +126,6 @@ IF[{- !$disabled{'ecx'} -}]
 GENERATE[x25519-x86_64.s]=asm/x25519-x86_64.pl
 GENERATE[x25519-ppc64.s]=asm/x25519-ppc64.pl
 ENDIF
+
+GENERATE[ecp_sm2p256-armv8.S]=asm/ecp_sm2p256-armv8.pl
+INCLUDE[ecp_sm2p256-armv8.o]=..
```

Comment:
Similarly to above, make it conditional on sm2 being enabled.

Reply:
fixed on https://github.com/openssl/openssl/commit/106d6f4b6a61d0e517eb1c6053e8222665baeb52